src/input/touch.c

```diff
@@ -24,6 +24,13 @@ static struct wlr_surface*
 touch_get_coords(struct seat *seat, struct wlr_touch *touch, double x, double y,
 		double *x_offset, double *y_offset)
 {
+	/* Fall out if mouse emulation is set */
```

Comment:
Suggestion: be a bit more verbose with this comment since why this enforces mouse emulation is not directly obvious. I did the same just now for tablets (https://github.com/labwc/labwc/pull/2255) ;)

Reply:
Fine - will look at your explanation and produce something similar!

Reply:
Done...